float32 pose_identification_number
